packages/lesswrong/lib/collections/comments/schema.ts

```diff
@@ -797,6 +799,7 @@ const schema: SchemaType<DbComment> = {
   },
 
   suggestForAlignmentUserIds: {
+    nullable: false,
```

Comment:
pretty sure this is redundant with this being a `arrayOfForeignKeysField` field (since that should now include `nullable: false` by default)